app/controllers/spree/product_questions_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Spree::ProductQuestionsController < Spree::StoreController
+class Spree::ProductQuestionsController < Spree::Api::V2::ResourceController
```

Comment:
this is not controller inside api folder, we cannot inherit from api controller

this is not the controller for api at all